DEVELOPMENT.md

```diff
@@ -4,9 +4,10 @@ This document outlines development practices that we follow while developing thi
 
 ## SwiftLint
 
-To run [SwiftLint](https://github.com/realm/SwiftLint) for all of the paths included in the `.swiftlint.yml`, run the following:
+Ensure that you have [SwiftLint](https://github.com/realm/SwiftLint) installed as we utilize it within our project.
 
+To install via [Homebrew](https://brew.sh/) run:
 ```
-bundle install
-bundle exec fastlane lint
+brew install swiftlint
```

Comment:
Can we use bundler instead to do this instead of needing everyone to also need homebrew? Since our Gemfile already has `fastlane` I think that contains `swiftlint`.

Reply:
Swiftlint [unfortunately isn't available via bundler using the Gemfile](https://github.com/realm/SwiftLint/issues/2093). 🙁  It looks like we _could_ use [this gem called Danger](https://rubygems.org/gems/danger-swiftlint/versions/0.26.0) if we want to stick to using bundler for all of these, but I'm not sure what other updates would be needed to use that gem instead.

Reply:
Swiftlint [unfortunately isn't available via bundler using the Gemfile](https://github.com/realm/SwiftLint/issues/2093). 🙁  It looks like we _could_ use [this gem called Danger](https://rubygems.org/gems/danger-swiftlint/versions/0.26.0) if we want to stick to using bundler for all of these, but I'm not sure what other updates would be needed to use that gem instead.

Reply:
Actually, looking into [Danger a bit further](https://github.com/ashfurrow/danger-ruby-swiftlint) it looks like you still need to do the following:

> SwiftLint also needs to be installed before you run Danger, which you can do via Homebrew or a Brewfile. 

So not sure that's really a viable solution either.

Reply:
Darn! Well thanks for investigating